### PR TITLE
According to task "Set up index.html to handle different modules #14"…

### PR DESCRIPTION
**Pull request by N0-N**
According to the task "Set up index.html to handle different modules" Made div tags for the header, announcements and calendar, etc., and Added .py files for the header, calendar, etc. and loaded these files into the <py-config> and <py-script> tags in index.html

### CODE REVIEW

File: code.py [hunk 1]
```diff
@@ -8,14 +8,4 @@ def build_element(type='div', class_name='', id='', content=''):
...
-print('Declan')
-print("!!!")
-print('!!!!!!!!!!!!!!')
-print('Taiki')
-  
-print('!!!!!')
-print('McKinley Evans')
-print('???')
-print('*****')
+  '''
```
Comment (mckinleyevans-wrdsb) [inline code.py:11]: 👍

File: index.html [hunk 2]
```diff
@@ -12,13 +12,27 @@
 
 <body>
   <div id='pyscript-code'>
+    <div id='pyscript-header'></div>
+    <div>Header</div>
```
Comment (mckinleyevans-wrdsb) [inline index.html:16]: Put this inside the `pyscript-header` div, not outside

File: index.html [hunk 3]
```diff
@@ -12,13 +12,27 @@
...
+      <div>Calendar</div>
+    </div>
   </div>
-  <div id='pyscript-announcements'></div>
+  
   <py-config>
       [[fetch]]
       files = ["./code.py"]
+      [[fetch]]
+      files = ["./Header.py"]
```
Comment (mckinleyevans-wrdsb) [inline index.html:29]: use lower-case for these names

File: index.html [hunk 4]
```diff
@@ -12,13 +12,27 @@
...
+      [[fetch]]
+      files = ["./Calendar.py"]
   </py-config>
   <py-script src="./code.py"></py-script>
+  <py-script src="./Header.py"></py-script>
```
Comment (mckinleyevans-wrdsb) [inline index.html:34]: again, lower-case

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (N0-N) [reply index.html:16]: Fixed

File: index.html (hunk 3 above)
Comment (N0-N) [reply index.html:29]: Fixed

File: index.html (hunk 4 above)
Comment (N0-N) [reply index.html:34]: Fixed

**Comment by N0-N** [comment]
Closing it due to it being the wrong branch.

**Pull request closed without merging**